### ISSUE TITLE
Error reported when loading pages on website

### ISSUE THREAD

**Issue by thjohu** [splitbrain/dokuwiki#2203]
Hello, 
 I have my website ajeko.org running for some years now but the pictures are not all loaded or only loaded partially. I found out that my website is using too much resources sometimes. Any idea how to correct?

 36022 /opt/cpanel/ea-php56/root/usr/bin/php-cgi /home/ajekoo1q/public_html/doku/doku.php 25% 20
 36056 /opt/cpanel/ea-php56/root/usr/bin/php-cgi /home/ajekoo1q/public_html/doku/doku.php 25% 16
 36059 /opt/cpanel/ea-php56/root/usr/bin/php-cgi /home/ajekoo1q/public_html/doku/doku.php 25% 16
 36065 /opt/cpanel/ea-php56/root/usr/bin/php-cgi /home/ajekoo1q/public_html/doku/doku.php 26% 16

 Thanx in advance

**Comment by splitbrain** [comment]
This report generally does not contain enough info to help you.

However because I had nothing else to do I had a look at your site and I noticed a couple of things:

* you have cross origin script requests that fail in Chrome (unrelated to this ticket)
* you have mixed content requests that fail in Chrome (unrelated to this ticket)
* your images sometime fail with status 508 (that's what this ticket is about)

From the headers I see that your server is running mod_bwlimited. Usually a Status 508 means "loop detected" but it seems it's also sometimes used as "resource limit error". So it seems sometimes some kind of resource limit is reached and images fail to load.

TL;DR your hoster is shit

**Comment by thjohu** [comment]
Hello

Thanx for you quick reply. I contacted the hoster and resources have been upgraded from 1 GB –> 3 GB, 1 vCPU –> 3vCPU. I still have the problem in Google and Microsoft Edge. I did a test with Mozella Firefox (I don’t use this browser only by exception) and everything runs smoothly. Is there something I can check to solve this annoying problem.
Thanx in advance

Hugo Smet

**Comment by thjohu** [comment]
Hello Andreas

It is finally solved at the hoster’s side.

Thanx and best wishes for 2018.

Kind regards
Hugo Smet
